lib/screens/v2/widget_instance/common/base_alert.ex

```diff
@@ -248,9 +248,15 @@ defmodule Screens.V2.WidgetInstance.Common.BaseAlert do
     |> MapSet.new(& &1.route_id)
   end
 
+  # Include a comment to define "informed routes": the routes affected by an alert that also exist at the current stop
+  #
+  # This is the updated version of "informed_routes" from alert.ex
+  # The other should be deleted
+  # Note: this does NOT reference downstream stops. Make sure that's fine and we don't need it
```

Comment:
+1 to checking on this, I vaguely remember the team deciding to stop showing any alerts besides those happening now and affecting the home stop in some way.

If that's true then we can get rid of a decent chunk of logic in this function as well as `location/1`. (`boundary_upstream` and `boundary_downstream` would no longer be applicable.)